app/recordtransfer/static/recordtransfer/js/base/index.js

```diff
@@ -1,10 +1,10 @@
-import "htmx-ext-head-support";
-import htmx from "htmx.org";
 import { setupMessages } from "./messages";
 import { setupNavbar } from "./navbar";
+import { setupHelpTooltips } from "./tooltip";
+
 
-window.htmx = htmx;
```

Comment:
Removing the `htmx` imports removes HTMX support in our application, can you add those lines back, and add back the line `window.htmx = htmx;`?